site/src/docs/patterns/cookies/common-helsinki-cookies.mdx

```diff
@@ -1,31 +1,30 @@
 ---
 slug: '/patterns/cookies/common-helsinki-cookies'
-title: 'Common Helsinki cookies'
-navTitle: 'Common Helsinki cookies'
+title: 'Commonly known Helsinki cookies'
+navTitle: 'Commonly known Helsinki cookies'
 ---
 
 import LeadParagraph from '../../../components/LeadParagraph';
 import ExternalLink from '../../../components/ExternalLink';
 import StatusLabel from '../../../components/StatusLabel';
 
-# Common Helsinki cookies
+# Commonly known Helsinki cookies
 
 <StatusLabel type="error">Draft</StatusLabel>
 
 <LeadParagraph>
-  Using common Helsinki cookies brings the benefit of consistent content across services that use the same components.
+  Using commonly Helsinki cookies brings the benefit of consistent content across services that use the same components.
 </LeadParagraph>
 
 ## Key principles of using common cookies
 
-- You can use common Helsinki cookies if your service is using a \*.hel.fi domain.
-- When common cookies are available, use them instead of creating custom cookies.
-- When a common cookie is used, it must be used **exactly** as described on this page. If you modify the cookie in any way, it is no longer a common cookie.
-- If your service is using custom cookies alongside common cookies, you still need to ask for consent for the custom cookies (if they are considered optional).
-- You can either use all or a part of the common cookies.
-- Note that some of the services (e.g., Matomo) may require multiple common cookies for it to work.
+- You can use commonly known Helsinki cookies if your service is using a \*.hel.fi domain.
+- When known cookies are available, use them instead of creating custom cookies. This way they are at least consistent even they are handled on service level.
+- If your service is using cookies, you need to ask for consent for the cookies (if they are considered optional).
+- You can either use all or a part of the commonly cookies.
```

Comment:
This common cookie concept is quite misleading, I think all "common cookies" references should be totally rewritten as now the text seems as if it would be recommended to put cookies to *.hel.fi domain on some parts.